aptos/scripts/init_wormhole.ts

```diff
@@ -0,0 +1,53 @@
+import { AptosAccount, TxnBuilderTypes, BCS, HexString, MaybeHexString, AptosClient, FaucetClient, AptosAccountObject } from "aptos";
```

Comment:
@optke3 we should remove this file. The chain can now be initialised by calling
```
aptos/scripts $ ./deploy
```
which sets up everything correctly, including the core bridge init stuff.